AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -653,6 +698,15 @@ - (void)displayImmediately
   [[self asyncLayer] displayImmediately];
 }
 
+- (void)recursivelyDisplayImmediately
```

Comment:
If I replace recursivelyDisplayImmediately with recursivelySetNeedsDisplay and recursivelyEnsureDisplay it doesn’t seem to complete synchronously, resulting in the UI flashing.